### PR TITLE
INT-7563: change types for disabledSources changes

### PR DESCRIPTION
**Pull request by RonaldEAM**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to
 
 ## Unreleased
 
+- Added `disabledSources` field to the `IntegrationInstance` type from the
```
Comment (dobregons) [inline CHANGELOG.md:12]: What will be the version with these changes?

File: packages/integration-sdk-core/src/types/context.ts [hunk 2]
```diff
@@ -36,6 +36,12 @@ export type IntegrationLoadExecutionConfigContext<
   instance: IntegrationInstance<TConfig>;
 };
 
+export type IntegrationInstanceExecutionContext<
+  TConfig extends IntegrationInstanceConfig = IntegrationInstanceConfig,
+> = ExecutionContext & {
+  instance?: IntegrationInstance<TConfig>;
```
Comment (dobregons) [inline packages/integration-sdk-core/src/types/context.ts:42]: 👍

File: packages/integration-sdk-core/src/types/instance.ts [hunk 3]
```diff
@@ -45,4 +45,13 @@ export interface IntegrationInstance<
...
    */
   config: TConfig;
+
+  /**
+   * A list of ingestion sources that are disabled for the integration instance.
+   */
+  disabledSources?: {
+    ingestionSourceId: string;
+    disabledOn: number;
+    disabledBy: string;
```
Comment (dobregons) [inline packages/integration-sdk-core/src/types/instance.ts:55]: These 2 should be optional.

File: CHANGELOG.md (hunk 1 above)
Comment (RonaldEAM) [reply CHANGELOG.md:12]: It will be v9.1.0, I'll add that in the release-x.y.z PR

**Pull request merged**